lib/spack/spack/test/data/config/config.yaml

```diff
@@ -14,6 +14,7 @@ config:
   checksum: true
   dirty: false
   concretizer: {0}
-  module_roots:
-    tcl: {1}
-    lmod: {2}
+#  module_roots:
+#    tcl: {1}
+#    lmod: {2}
+#
```

Comment:
I think this should just be deleted.  That said, I think we need code that translates the old module format into the new one, and we should have a test for that to ensure that it works.